impl_test.go

```diff
@@ -196,9 +237,25 @@ func TestFuncs(t *testing.T) {
 			t.Errorf("funcs(%q).err=%v want %s", tt.iface, err, errBool(tt.wantErr))
 			continue
 		}
-		if !reflect.DeepEqual(fns, tt.want) {
-			t.Errorf("funcs(%q).fns=\n%v\nwant\n%v\n", tt.iface, fns, tt.want)
+
+		if tt.ignoreComments {
+			if len(fns) != len(tt.want) {
+				t.Errorf("funcs(%q).fns=\n%v\nwant\n%v\n", tt.iface, fns, tt.want)
+			}
+			for i, fn := range fns {
+				if fn.Name != tt.want[i].Name ||
+					!reflect.DeepEqual(fn.Params, tt.want[i].Params) ||
+					!reflect.DeepEqual(fn.Res, tt.want[i].Res) {
+
+					t.Errorf("funcs(%q).fns=\n%v\nwant\n%v\n", tt.iface, fns, tt.want)
+				}
+			}
```

Comment:
Put a continue here, remove the else, add a comment like `// check everything including comments`, and unindent the following lines.

Reply:
Done.